specs/capri/Multi-attach.md

```diff
@@ -0,0 +1,136 @@
+# Title
+
+**Author(s)**: [Jack Li](https://github.com/jackhaibo)
+
+## Summary
+
+This design is proposed to enable multiAttach of volumes.
+
+## Motivation
+
+Currently we refuse to attach a volume that is already attached (in-use). There are situations where a cloud user may have deployed a clustered file-system on a Cinder volume and they would like the ability to attach a volume to multiple hosts/servers at the same time.
+
+### Goals
+
+This spec proposes a set of changes to enable and control the use of multiAttach volumes from the OpenSDS side.
+
+## Design Details
+
+1. The drivers that support multi-attach ability need to report this in their capabilities through pool configuration file. The multi-attach volume creation request will select multi-attach enabled pool from pools which come from different drivers that many of them may not support this ability. 
```

Comment:
s/different drivers that many of them may not support this ability/different drivers and many of them may not support this ability.

Reply:
Done.